src/monitor-view-events.js

```diff
@@ -9,25 +9,32 @@ import { triggerMethodOn, triggerMethodMany } from './trigger-method';
 // whenever a rendered view is attached or an attached view is rendered.
 function MonitorViewEvents(view) {
   if (view._areViewEventsMonitored) { return; }
+
+  if (!view._getImmediateChildren) {
+    return;
+  }
```

Comment:
Ah.. because it's added in the region's show in addition to the view constructor.. that makes sense.